test/unit/targeting_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_plugin_helper'
+
+describe Targeting do
+  let(:targeting) { FactoryGirl.build(:targeting) }
+  let(:bookmark) { bookmarks(:one) }
+  let(:host) { FactoryGirl.create(:host) }
+
+  context 'Able to be created with search term' do
+    before { targeting.search_query = "name = foo" }
+    it { assert targeting.save }
```

Comment:
same comments for the whole file, I'd prefer using `assert_valid` and `.valid?` instead of doing real save